cohttp-eio/examples/client_tls.ml

```diff
@@ -9,7 +9,11 @@ let null_auth ?ip:_ ~host:_ _ =
   Ok None (* Warning: use a real authenticator in your code! *)
 
 let https ~authenticator =
-  let tls_config = Tls.Config.client ~authenticator () in
+  let tls_config =
+    match Tls.Config.client ~authenticator () with
+    | Error (`Msg msg) -> failwith ("tls configuration problem: " ^ msg)
+    | Ok tls_config -> tls_config
```

Comment:
Can you fix this on the nix ci?

Reply:
Hmm I think we need to wait for `tls.1.0.0` from https://github.com/nix-ocaml/nix-overlays/pull/1568 and the `conduit.7.0.0` release https://github.com/ocaml/opam-repository/pull/26454 to be merged and updated on nix-ocaml? (unless you prefer a follow-up PR once both are available on nix-ocaml?)

Reply:
I use nix to develop this repo so I would prefer to wait a bit if possible. If the changes don't propagate within a week, we can merge this PR.

Reply:
Yes I agree we should wait, the CI is still unhappy atm :)